src/google/protobuf/port_def.inc

```diff
@@ -308,6 +308,15 @@
 
 #if defined(GOOGLE_PROTOBUF_NO_RTTI) && GOOGLE_PROTOBUF_NO_RTTI
 #define PROTOBUF_RTTI 0
+#elif defined(__has_feature)
+// https://clang.llvm.org/docs/LanguageExtensions.html#has-feature-and-has-extension
+#define PROTOBUF_RTTI __has_feature(cxx_rtti)
+#elif !defined(__cxx_rtti) && __cplusplus >= 199711L
```

Comment:
I think you can leave out the `__cplusplus` check since we require C++11 anyway.

Reply:
Indeed, done